src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -738,8 +738,8 @@ public static explicit operator Half(float value)
             const uint SingleBiasedExponentMask = float.BiasedExponentMask;
             // Exponent displacement #2
             const uint Exponent13 = 0x0680_0000u;
-            // Maximum value that is not Infinity in Half
-            const float MaxHalfValueBelowInfinity = 65520.0f;
+            // The value above Half.MaxValue
+            const float HalfAboveMaxValue = 65520.0f;
```

Comment:
nit: I'm not sure this name change is an improvement The other name wasn't great, but I think it was a little more clear.

In particular this is the maximum infinitely precise value that will round down to `MaxValue` rather than rounding up to `PositiveInfinity`

Reply:
This comment is still pending.